### ISSUE TITLE
Remove symbol in the "Locate symbol" list by clicking again on its symbol in the expression table

### ISSUE THREAD

**Issue by charles-plessy** [Hypercubed/DEIVA#14]
Hi Jayson and Anton,

at the moment it is possible to click on a gene symbol displayed in the expression table, to add it to the _Locate symbol_ list and make it appear on the plot.  Just after the user did that, the mouse pointer is still on the symbol in the expression table.

Now let's imagine that after seeing the position of the dots representing the symbol in the plot, the user concludes that it is not so interesting and wants to remove it. It would be convenient if he would only have to click again on the mouse button to do this.

In other words, I am proposing that when clicking on a symbol that is already selected for display, this un-selects it.

**Comment by Hypercubed** [comment]
Good idea... this will be in the next version.  I'm also considering how to indicate the symbol is currently selected or not.  Note that a feature can have multiple symbols.

**Comment by Hypercubed** [comment]
Publish in latest version.  Thanks!
